src/index.html

```diff
@@ -6,9 +6,79 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Kosugi&family=Roboto:wght@300;500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html" class="header__logo">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="list__item">
+            <a
+            class="nav__link is-active"
+            href="#apple">
+            apple
+            </a>
+          </li>
+          <li class="list__item">
+            <a
+            class="nav__link"
+            href="#samsung">
+            samsung
+            </a>
+          </li>
+          <li class="list__item">
+            <a
+            class="nav__link"
+            href="#smartphones">
+            smartphones
+            </a>
+          </li>
+          <li class="list__item">
+            <a
+            class="nav__link"
+            data-qa="hover"
+            href="#laptops & computers">
+            laptops & computers
+            </a>
```

Comment:
```suggestion
            <a
              class="nav__link"
              data-qa="hover"
              href="#laptops & computers"
            >
              Laptops & computers
            </a>
```

Reply:
<img width="1011" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/86789161/ac87ab3a-f4d8-400d-8028-9482dd161fc1">
Pay attention to formatting 😉 
Check all code for similar problems